dataplicity/tools.py

```diff
@@ -1,18 +0,0 @@
-from __future__ import unicode_literals
```

Comment:
This was used to potentially support other schemes for storing serial / auth. Deprecated now.